make/migrations/postgresql/0030_1.11.0_schema.up.sql

```diff
@@ -43,6 +43,19 @@ CREATE TABLE artifact_reference
   CONSTRAINT  unique_reference UNIQUE (parent_id, child_id)
 );
 
+/* artifact_trash records deleted artifact */
+CREATE TABLE artifact_trash
+(
+  id            SERIAL PRIMARY KEY NOT NULL,
+  artifact_id   int NOT NULL,
+  media_type    varchar(255) NOT NULL,
+  manifest_media_type varchar(255) NOT NULL,
+  project_id    int NOT NULL,
+  repository_id int NOT NULL,
```

Comment:
As the project and repository can be deleted before running GC, so seems we need to record the full repository name(project_name/repository_name) here

Reply:
I am think to use get the name in the GC job, but it's ok to set the name here.

Reply:
update to repository name